### PR TITLE
Update contributing guidelines

### PR DESCRIPTION
**Pull request by ncudlenco**
* Updated the commit types
* Updated the collaborative models for external and internal contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Thanks for your interest in contributing to Büchi! :tada: We value everybody's
 
 ## Contribute! (Pull Request) 
 
-Before you start, we strongly recommend you to search on existing PR's and issues to see if the issue has already been reported.
+Before you start, we strongly recommend searching for existing PRs and issues to see if the issue has already been reported.
+
+For external collaborators, we are adhering to the [Fork and pull collaboration model]([url](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#fork-and-pull-model))
```
Comment (Immich) [inline README.md:9]:
```suggestion
**For external collaborators**, we are adhering to the [Fork and pull collaboration model]([url](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#fork-and-pull-model))
```

File: README.md [hunk 2]
```diff
@@ -37,35 +39,75 @@ $ git fetch upstream
...
+* Push the changes to your account
 
 ```
 $ git push -u origin <changes-name>
 ```
 
-* Get a PR merged! :1st_place_medal:
+* Get a PR merged! :1st_place_medal: ([click here for more details about creating a PR from a fork]([url](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)))
+
+For internal collaborators we are using the [Shared repository collaboration model]([url](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#shared-repository-model))
```
Comment (Immich) [inline README.md:50]:
```suggestion
**For internal collaborators**, we are using the [Shared repository collaboration model]([url](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#shared-repository-model))
```

**Review by algorithmich — CHANGES_REQUESTED** [review]
Good job! I would add a table of contents and take into account the suggestions I made :)

**Comment by ncudlenco** [comment]
> Good job! I would add a table of contents and take into account the suggestions I made :)

Done

**Review by algorithmich — CHANGES_REQUESTED** [review]

**Pull request merged**